### ISSUE TITLE
getussd: used wrong encoding

### ISSUE THREAD

**Issue by jazzl0ver** [gammu/gammu#179]
Gammu sets encoding to HEX which leads to "UNKNOWN response". Here is the transcript from the log - http://pastebin.com/BxfEseM3

Here is the manual session from minicom:
```
AT+CSCS?
+CSCS: "IRA"

OK
AT+CUSD=1,"*100#"
+CUSD: 0,"799.55@.+1000@ 70 25@=K5 >B25BK =0 2>?@>AK *5                        2

OK
```

And here's what's happening after setting the encoding to HEX:
```AT+CSCS="HEX"
OK
AT+CUSD=1,"*100#"
+CUSD: 2
```

Can you please fix it?

**Comment by nijel** [comment]
Attaching log from pastebin: 
[gammu.log.txt](https://github.com/gammu/gammu/files/111271/gammu.log.txt)

**Comment by nijel** [comment]
What phone/modem is that? Most likely it just needs F_ENCODED_USSD enabled...

**Comment by jazzl0ver** [comment]
It's a gsm router sprut universal (http://gsm-sprut.com/p.php?pid=240&l=1; sorry, there's no english page).
how to enable that?

**Comment by Docbroke** [comment]
In my case it is

DELL DW5560 WWAN Mobile Broadband HSPA MiniPCI Express Card

**Comment by Docbroke** [comment]
In my case it is

DELL DW5560 WWAN Mobile Broadband HSPA MiniPCI Express Card

**Comment by nijel** [comment]
Can you try if it now works better in Git?

@Docbroke Please report separate issue for that if it's different device.

**Comment by jazzl0ver** [comment]
nijel, it works weird. here is the reply I'm getting when using minicom:
```
AT+CUSD=1,"*105*1#"
+CUSD: 0,"Summa dostupny'x sredstv 209.67 rub.",0
```
and here is a reply using the latest gammu:
```
$ pwd
/home/test/gammu/build-configure/gammu
[test@oreon gammu]$ ./gammu getussd "*105*1#"
Press Ctrl+C to break...
USSD received
Status               : No action needed
Service reply        : ""
Operation aborted.
```
and this is the output of the old gammu (1.17.0):
```
$ gammu getussd "*105*1#"
Press Ctrl+C to break...
USSD received
Status               : No action needed
Service reply        : "Nepravilnyi nomer uslugi"
```
Log files (the old and the new) are attached.

**Comment by Docbroke** [comment]
@nijel I did created issue #178, but that thread is closed as issue is similar to this one

**Comment by nijel** [comment]
Can you please provide debug log from the git version?

**Comment by Docbroke** [comment]
log with gammu 1.37.0

http://pastie.org/10714602

**Comment by nijel** [comment]
@Docbroke what was problem there? It seems the device stopped responding...

**Comment by nijel** [comment]
Nevermind, I've found the problem, should be fixed now.

**Comment by raoulsigne** [comment]
Hi to all, i'm still having trouble to get used response with gamma; I'm using version 1.37.4 with Huawei E169

Thanks for help

**Comment by raoulsigne** [comment]
Hello Now USSD is working on pico com correctly but not with gammu but I need it to work with gammu

**Comment by nijel** [comment]
@raoulsigne The above mentioned fix is part of 1.37.90 and later, so please upgrade.
